client/dist/index.css

```diff
@@ -45,4 +45,22 @@ q:before, q:after {
 table {
 	border-collapse: collapse;
 	border-spacing: 0;
-}
\ No newline at end of file
+}
```

Comment:
This all get moved into the style- components?

Reply:
yes let me delete that!